osu.Game/Screens/OnlinePlay/Multiplayer/Match/MultiplayerReadyButton.cs

```diff
@@ -168,7 +168,7 @@ public override LocalisableString TooltipText
         {
             get
             {
-                if (room?.Countdown != null && multiplayerClient.IsHost && multiplayerClient.LocalUser?.State == MultiplayerUserState.Ready)
+                if (room?.Countdown != null && multiplayerClient.IsHost && multiplayerClient.LocalUser?.State == MultiplayerUserState.Ready && room.Settings.AutoStartDuration == TimeSpan.Zero)
```

Comment:
Thinking that a future addition will be an "extend countdown" or "reset countdown" in place of this, for auto start.

Reply:
I would hope the interface is updated by that point, because this button's overloaded with functionality. Or maybe the `CountdownButton` can be reused for this case.